scripts/test-check-packages.sh

```diff
@@ -121,11 +119,6 @@ for d in test/packages/${PACKAGE_TEST_TYPE:-other}/${PACKAGE_UNDER_TEST:-*}/; do
     elif [ "${PACKAGE_TEST_TYPE:-other}" == "with-logstash" ] && [ "${package_to_test}" == "system_benchmark" ]; then
         elastic-package benchmark system --benchmark logs-benchmark -v --defer-cleanup 1s
     else
-      if [[ "${ELASTIC_PACKAGE_TEST_ENABLE_INDEPENDENT_AGENT}" == "false" && "${package_to_test}" == "auditd_manager_independent_agent" ]]; then
```

Comment:
Umm, this `auditd_manager_independent_agent` package could be moved out of the `with-custom-agent` directory, right? And then maybe renamed back to `auditd_manager`.

Reply:
Ok, moving this package out of this folder :+1: